doc/main.html

```diff
@@ -3916,13 +3858,13 @@ <h4>Registered Disclosure Documents</h4>
           </p>
 
           <p>
-            The submitting entity also may publish an <a>amendment</a>
+            The submitting entity also may publish a <a>revision</a>
             to the document, provided it is promptly submitted to
-            the <a>Society</a> for processing as an addition to or
-            replacement of a portion of the <a>Registered Disclosure
+            the <a>Society</a> for processing as a replacement of a 
+            portion of the <a>Registered Disclosure
```

Comment:
This is confusing, should remove "portion" as that sounds like an amendment. 
```suggestion
          <a>Registered Disclosure
```